boards/FSM/Test/Src/Test_FlowMeter.cpp

```diff
@@ -0,0 +1,30 @@
+#include "Test_Fsm.h"
+
+FAKE_VALUE_FUNC(float, get_flow_rate);
+
+class FlowMeterTest : public testing::Test
+{
+  protected:
+    struct FlowMeter *flow_meter;
```

Comment:
nit'; convention so far is to but variables below the functions (i.e. Setup and TearDown)

Reply:
done